### PR TITLE
support multiple processors

### PR DESCRIPTION
**Pull request by maggiepint**

### CODE REVIEW

File: lib/fileProcessor.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+const moment = require('moment');
+class FileProcessor {
+  constructor(store, files) {
```
Comment (maggiepint) [inline lib/fileProcessor.js]: Is this change on your local machine? I don't see a logger at line 14 of gitHubProcessor in develop.

File: lib/fileProcessor.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    this.store = store;
```
Comment (maggiepint) [inline lib/fileProcessor.js]: No, but this creates consistency with the constructor for gitHubProcessor, which seems logical. I can remove it though.

File: lib/fileProcessor.js (hunk 1 above)
Comment (jeffmcaffer) [inline lib/fileProcessor.js]: hmmm, honestly don't know what I was looking at when I wrote that comment but do recall it was on GitHub in some file.  You are right, the GitHubProcessor constructor does not have options in the  signature.  Ignore my comment.

File: lib/fileProcessor.js (hunk 2 above)
Comment (jeffmcaffer) [inline lib/fileProcessor.js]: suggest removing for clarity and simplicity.  In this case each processor needs its own set of data.  The FileProcessor needs the file list but not a store.  The GitHubProcessor needs a store, ...

**Comment by jeffmcaffer** [comment]
obsolete. Closing

**Pull request closed without merging**